src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonNavLink } from '../PersonNavlink';
+
+type Props = {
+  people: Person[],
+  person: Person,
+  selectedSlug: string,
+};
+
+export const PersonItem: FC<Props> = ({
+  people,
+  person,
+  selectedSlug,
+}) => {
+  const isSelectedPerson = person.slug === selectedSlug;
+  const personMotherName = person.motherName || '-';
+  const personFatherName = person.fatherName || '-';
+
+  const personMother = people
```

Comment:
will be better to prepare the array (add mother and father objects to each person) after retrieving people from server instead of on each component rendering